src/collective/glossary/browser/static/main.js

```diff
@@ -1,6 +1,6 @@
 $(function(){
   $('#content-core').glossarizer({
-    sourceURL: '@@glossary',
+    sourceURL: portal_url + '/@@glossary',
```

Comment:
Good question, I just found this variable (I saw it in another file one day).

After a little search I found where it is https://github.com/plone/Products.CMFPlone/blob/4.3.x/Products/CMFPlone/browser/configure.zcml#L31-L36

It generates these lines (inserted into jsregistry):

    /* - plone_javascript_variables.js - */
    var portal_url = 'http://localhost:8080/Plone';
    var form_modified_message = 'Your form has not been saved. All changes you have made will be lost.';
    var form_resubmit_message = 'You already clicked the submit button. Do you really want to submit this form again?';
    var external_links_open_new_window = 'false';
    var mark_special_links = 'false';
    var ajax_noresponse_message = 'No response from server. Please try again later.';

It looks that with Plone 5 things will change, this browser view don't exists at master branch of `Products.CMFPlone`